crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -444,23 +457,25 @@ impl<'w, 's> Commands<'w, 's> {
     #[inline]
     #[track_caller]
     pub fn entity(&mut self, entity: Entity) -> EntityCommands {
-        #[inline(never)]
-        #[cold]
-        #[track_caller]
```

Comment:
Some of the perf regressions are probably from losing these annotations. We should try to add them back.

Reply:
Good catch, that would do it.

Reply:
I did smaller benchmarks before and after making that change (only the `insert_commands` benchmark, because that had been the most volatile) and didn't see any difference. We're aiming to remove the panic from `entity` anyway, I'm not sure there's a place for those